include/dgl/base_heterograph.h

```diff
@@ -368,6 +368,27 @@ class BaseHeteroGraph : public runtime::Object {
    */
   virtual SparseFormat SelectFormat(dgl_type_t etype, SparseFormat preferred_format) const = 0;
 
+  /*!
+   * \brief Get the sparse format of the graph.
+   * 
+   * This function is only implemented for UnitGraph.
+   * \return a string representing the sparse format: 'coo'/'csr'/'csc'/'any'
+   */
```

Comment:
The docstring should be self-contained. No need to mention UnitGraph. Also, why not return `SparseFormat` object?